### PR TITLE
chore: react native 0.59.8

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Bumping from React Native 0.55.4 to latest (0.59.8)

Tasks:
- [x] Bump RN, Jest, React, React-Native-Web, react-native-test-renderer to latest
- [x] Bump Babel-Jest
- [x] Update metro configs for app builds
- [x] Update invalid accessibilityRoles as those are throwing errors on latest react-native
- [x] Update gradle configuration on the android-app library bundle
- [x] Update gradle configuration on the android storybook bundle
- [x] Update metro configs for native storybooks
- [x] Fix React-Native-Web issues
- [x] Fix ios storybook issues 
- [ ] Integrate to the existing ios app (Need iOS dev assistance on this one)
- [x] Fix broken tests (Jest update has some breaking changes, this might take time)

### CODE REVIEW

File: packages/article-paragraph/src/article-paragraph.web.js [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Paragraph } from "./styles/responsive";
-import styles from "./styles";
 
-const BodyParagraph = ({ children }) => (
-  <Paragraph style={styles.articleTextElement}>{children}</Paragraph>
-);
+const BodyParagraph = ({ children }) => <Paragraph>{children}</Paragraph>;
```
Comment (nathggns) [inline packages/article-paragraph/src/article-paragraph.web.js]: Not worth changing in this PR but this component is now redudant.

File: packages/author-profile/__tests__/shared-with-style.native.js [hunk 2]
```diff
@@ -33,34 +29,5 @@ export default () => {
     slug: "some-slug"
   };
 
-  const tests = [
-    {
-      name: "an article list header faded in",
```
Comment (nathggns) [inline packages/author-profile/__tests__/shared-with-style.native.js]: Why is this test being deleted?

Comment (tuncaulubilge) [reply packages/author-profile/__tests__/shared-with-style.native.js]: It was breaking and wasn't really a meaningful test tbh

**Comment by parkjh** [comment]
@tuncaulubilge yea the react article crashes for me on local render with this change included...

**Comment by tuncaulubilge** [comment]
@parkjh I managed to reproduce the issue with render, it's caused due to `userState` missing on render, which will be fixed with Aron's in progress PR. The rest looks all good to me in render

**Pull request merged**